base/special/expm1.jl

```diff
@@ -0,0 +1,238 @@
+# ====================================================
```

Comment:
Probably the comment should start with:
```jl
# This code is a Julia translation of the C code from ... with the following license:
```
to make it clear that this is not the original Sun code.

Reply:
Also, see https://github.com/JuliaLang/julia/blob/master/base/special/exp.jl